data/hai/unfettered 2 war preparations.txt

```diff
@@ -43,12 +113,12 @@ mission "<uhai> 10: Belt Materials"
 			label asteroidtuto
 			action
 				set "unfettered: asteroid tuto"
-			`	She seems a bit surprised, "You've never mined asteroids? I recommend you install an asteroid scanner so your ship can target them automatically. Just try to keep your ship stable around one and shoot it as you would any other ship. Keep in mind they have no shields, and you might not want to waste missiles on them, since they don't shoot back."`
+			`	She seems a bit surprised, "You've never mined asteroids? I recommend you install an asteroid scanner so your ship can target them automatically. Just try to keep your ship stable around one and shoot it as you would any other ship. Keep in mind they have no shields, and you might not want to waste missiles on them, since they don't shoot back. One thing everyone may not know is that once you've exhausted all the asteroids in a system, you may as well land or change system knowing they'll be back the next time you launch!"`
```

Comment:
```suggestion
			`	She seems a bit surprised, "You've never mined asteroids? I recommend you install an asteroid scanner so your ship can target them automatically. Just try to keep your ship stable around one and shoot it as you would any other ship. Keep in mind they have no shields, and you might not want to waste missiles on them, since they don't shoot back. If you ever run out of targets, you can fly around for a couple of days and come back; new asteroids will likely enter the system in that time."`
```